tests/acceptance/ready-set-go-test.js

```diff
@@ -44,3 +44,35 @@ test('passes the activity', function(assert) {
     assert.equal(page.activityCompletedMessage, 'Next activity');
   });
 });
+
+test('restart activity', function(assert){
+  new Pretender(function(){
+    this.get('/api/activities.json', function() {
+      return [200, {"Content-Type": "application/json"}, JSON.stringify(ACTIVITIES)];
+    });
+  });
+
+  page.visit();
+
+  andThen(function(){
+    assert.equal(currentURL(), '/activities/1');
+    assert.equal(currentRouteName(), 'activity');
+    assert.equal(page.completed, '');
+    assert.equal(page.pending, 'abcd');
+  });
```

Comment:
I wouldn't test this initial state here, there's another test that already test this. I would do something like

```js
page.visit().typeLetter('a').click('restart');
```

and then test that the the page.completed is empty and pending is full.

What do you think?